vnu-jar.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
-
-module.exports = require('path').join( __dirname, 'build/dist/vnu.jar' );
+const { execFile } = require('child_process');
+const path = require('path');
+const { resolveJava } = require('./vnu-java-resolver');
```

Comment:
Consider importing this within `check`, so all the path traversals, JS imports can be skipped for the non-`check` use cases.